### ISSUE TITLE
Add golangci-lint

### ISSUE THREAD

**Issue by rusinikita** [rusinikita/trainer#7]
Set config from https://github.com/rusinikita/devex/blob/main/.golangci.yaml

https://golangci-lint.run/

**Comment by mortum5** [comment]
@rusinikita Do you want just add lint config or fix all suggestions?

**Comment by rusinikita** [comment]
Yes. Fix suggestions. It's not fast issue. Consider to focus #10 or #2
